recipes/perl-atlas-modules/build.sh

```diff
@@ -26,23 +26,21 @@ fi
 
 cpanm -l $PERLLIB MooseX::FollowPBP \
                     URI::Escape \
-                    URL::Encode \
                     Config::YAML \
                     File::Basename \
                     Bio::MAGETAB \
                     Date::Parse \
                     Test::MockObject \
                     Text::TabularDisplay \
-                    Log::Dispatch::File \
-                    IO::CaptureOutput \
                     TIMB/DBI-1.636.tar.gz \
                     MIYAGAWA/Class-Trigger-0.15.tar.gz \
                     TMTM/Class-DBI-v3.0.17.tar.gz \
                     DETI/Proc/Proc-Daemon-0.14.tar.gz \
```

Comment:
I would try un pining these and possibly move them to `package::name` usage (to un pin them).